cypress/tests/mocked/parallel-2/cancelSupporterPlus.cy.ts

```diff
@@ -123,7 +123,7 @@ describe('Cancel Supporter Plus', () => {
 		cy.wait('@get_cancelled_product');
 
 		cy.findByRole('heading', {
-			name: 'Monthly support + extras cancelled',
+			name: 'Your All-access digital subscription is cancelled',
```

Comment:
Elsewhere, `all-access` is all lowercase (for example `Change to all-access digital`) - should we be consistent?`</nitpick>`

Reply:
yep, you're right to nitpick!